cartridge/graphql.lua

```diff
@@ -346,6 +367,42 @@ local function init(httpd)
     )
 end
 
+--- Set up trigger for GraphQL handlers.
+--
+-- It will be executed **before** top-level resolvers, which were
+-- registered using `add_callback` or `add_mutation` methods.
+--
+-- The trigger function is called with one argument:
+-- `{operation = 'query|mutation', field = '[prefix.]field_name'}`
+--
+-- If the parameters are `(nil, old_trigger)`, then the old trigger is
+-- deleted.
+--
+-- (**Added** in v2.0.1-52)
+--
+-- @usage
+--    local function log_request(req)
+--        log.info('GraphQL %s: %s', req.operation:upper(), req.field)
+--        -- Will print "GraphQL QUERY cluster.auth_params"
+--    end)
+--
+--    graphql.on_resolve(log_request) -- start logging
+--    graphql.on_resolve(nil, log_request) -- stop logging
+--
```

Comment:
graphql.on_resolve() - should return list of triggers

If we want to respect tarantool triggers flow

Reply:
I thought about it, but decided not to overload the patch without necessity. Also I don't like tarantool triggers flow because it's return values are ambiguous.